pkg/server/test/check.go

```diff
@@ -137,7 +382,7 @@ func BenchmarkCheck(b *testing.B, ds storage.OpenFGADatastore) {
 
 		checkQuery := commands.NewCheckCommand(
 			ds,
-			bm.checker,
+			graph.NewLocalChecker(),
 			typeSystem,
 			commands.WithCheckCommandMaxConcurrentReads(config.DefaultMaxConcurrentReadsForCheck),
```

Comment:
`config.DefaultMaxConcurrentReadsForCheck` is `math.MaxUint32`. This can overwhelm the server on some queries (e.g. `with_complex_ttu`). Let's make this smaller, e.g. 50 :)